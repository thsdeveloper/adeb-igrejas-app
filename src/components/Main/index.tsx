import * as S from './styles'

const Main = () => (
  <S.Wrapper>
    <S.Logo src="/img/logo.svg" alt="Logo da AdebSystem" />
    <S.Illustration src="/img/undraw_my_current_location_om7g.svg" />
    <S.Title>Adebsystem</S.Title>
    <S.Description>App para localização das igrejas</S.Description>
  </S.Wrapper>
)

export default Main
