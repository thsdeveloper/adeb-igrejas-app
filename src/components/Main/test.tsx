import { render, screen } from '@testing-library/react'

import Main from '.'

describe('<Main />', () => {
  it('Deve renderizar o título', () => {
    const { container } = render(<Main />)

    expect(
      screen.getByRole('heading', { name: /react avadddddddnçado/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
