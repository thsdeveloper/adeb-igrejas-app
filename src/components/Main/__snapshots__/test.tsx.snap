// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> Deve renderizar o título 1`] = `
<main>
  <h1>
    React Avançado
  </h1>
</main>
`;
